marketing_crm_partner/README.rst

```diff
@@ -0,0 +1,87 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Tracking Fields in Partners
+===========================
+
+This module extends the functionality of the CRM to support having the tracking
+fields available in the partner and copy them there automatically when the
+partner is created from a lead/opportunity.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to *Sales > Leads & Opportunities*.
+* Configure the available sources and campaigns from the menus there.
```

Comment:
I don't find any place to configure this
![26-02-2016 11-26-26](https://cloud.githubusercontent.com/assets/8736623/13349695/ca820c4e-dc7b-11e5-8d39-6c59aef99052.png)